### ISSUE TITLE
Neatness of UG

### ISSUE THREAD

**Issue by LuYiting0913** [LuYiting0913/pe#17]
![image.png](https://raw.githubusercontent.com/LuYiting0913/pe/main/files/a7a348d2-3cbb-41e0-94a7-5dfb2a70fd03.png)

no/minimal indentation for sub sections. hard to follow

**Comment by nus-pe-script** [comment]
# Team's Response
We believe that this does not affect the readability of the UG and the user can simply scroll down to continue reading.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

-------------------
